Revert default WebsiteDataStore change

Causes incognito mode to fail

See:
   https://bugs.webkit.org/show_bug.cgi?id=216430

--- webkitgtk-2.30.0/Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp	2020-09-04 07:24:07.000000000 +0000
+++ webkitgtk-2.30.0/Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp	2020-09-15 17:53:16.919718148 +0000
@@ -1874,8 +1874,7 @@
 WebKitDownload* webkitWebContextStartDownload(WebKitWebContext* context, const char* uri, WebPageProxy* initiatingPage)
 {
     WebCore::ResourceRequest request(String::fromUTF8(uri));
-    auto& websiteDataStore = webkitWebsiteDataManagerGetDataStore(context->priv->websiteDataManager.get());
-    return webkitWebContextGetOrCreateDownload(&context->priv->processPool->download(websiteDataStore, initiatingPage, request));
+    return webkitWebContextGetOrCreateDownload(&context->priv->processPool->download(WebKit::WebsiteDataStore::defaultDataStore().get(), initiatingPage, request));
 }
 
 void webkitWebContextRemoveDownload(DownloadProxy* downloadProxy)
